docs/depfile.md

```diff
@@ -202,13 +211,13 @@ Example:
 parameters:
```

Comment:
Another reminder to myself, my current attempt at checking for the new file format won't work if the old file already has parameters. I will have to explicitly check for all parameters at both locations.